integration/data/conftest.py

```diff
@@ -49,6 +50,38 @@ def grpc_controller_no_frontend(request, grpc_controller_client):
                                   frontend="")
 
 
+@pytest.fixture
```

Comment:
Move those two fixtures to `test_basic_ops.py` (if possible) since it's unlikely they will be shared by others.

Reply:
I certainly can move them, but it looks like we always use the conftest.py for fixtures.
By moving them we are breaking convention, do you want me to still move them?

Reply:
You have a point. If we moved them, the other contributors might get confused on where to put the fixtures. Keep them here then.